### ISSUE TITLE
In the new versions of rasa-core the interactive learning tables are not printed correctly

### ISSUE THREAD

**Issue by paulaWesselmann** [RasaHQ/rasa_nlu#3062]
Instead of the nice table with the chat history, tn interactive learning the tables now look like this:
```
Chat History
b' #    \x1b[94mBot      \x1b[39m 
 \x1b[91mYou       \x1b[39m 
\n\x1b(0qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq\x1b(B\n 1 
\x1b[96maction_listen\x1b[39m                                                                 \n\x1b(0qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq\x1b(B\n 2
\x1b[91mhello mate\x1b[39m \n
intent: greet 0.86 \n\x1b(0qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq\x1b(B\n 3    \x1b[96mutter_name\x1b[97m 1.00\x1b[39m                                                                          \n      \x1b[94mHey there! Tell me your name.\x1b[39
\n      \x1b[96maction_listen\x1b[97m 1.00\x1b[39m                                                                       \n\x1b(0qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq\x1b(B\n 4
\x1b[91mMy name is [Isabella](name:isabella)\x1b[39m \n                                                                             intent: name 0.87 
\n\x1b(0qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq\x1b(B\n 5    slot{"name": "isabella"}                                                                 \n      \x1b[96mutter_greet\x1b[39m
\n      \x1b[94mNice to you meet you isabella. How can I help?\x1b[39m                                        
Current slots:
	age: None, name: isabella, zip: None
```

**Comment by tmbo** [comment]
I think I fixed this after the merge, undoing the change done by a PR trying to fix #3091 

Is this still an issue?

**Comment by paulaWesselmann** [comment]
@tmbo I just tested IL in the merged version and it's fixed! I'll close this issue.
